### DISCUSSION TITLE
How to construct MWE for coc ?

### DISCUSSION THREAD

**Discussion by iago-lito** [neoclide/coc.nvim#4176] [Q&A]
I wish to report some coloring problem to Coc, but I first need to construct a minimal working example to reproduce it.

I'm having trouble doing so, because it actually is nontrivial (need a plugin manager, node, the presumably-faulty coc-plugin..) and I don't find help in this respect within coc documentation.

Can the wiki, or the README, provide a stump for constructing MWE featuring coc? This way I could be able to `[n]vim -u coc-mwe.vim my-buggy-file` and either understand the problem myself or produce a helpful bug report ?

**Comment by chemzqm** [comment]
What does MWE means?

**Reply by fannheyward** [reply]
Minimal working example?

**Reply by iago-lito** [reply]
Yes, Minimal Working Example (sorry for that). A MWE useful to report bugs would be something bundling:
- Minimal `.vim/` folder with up-to-date Coc inside.
- Minimal `.vimrc` configuration file loading Coc and the faulty Coc module.
- Minimal file to reproduce the problem within.

**Reply by chemzqm** [reply]
A minimal vimrc is enough most of the time.

**Reply by iago-lito** [reply]
The problem is that I'm not sure what to put in this vimrc so that  Coc is up and ready to reproduce when I do `nvim -u .minimal_vimrc fault_file`. Do I need to (re-)install it from this vanilla vim session?

**Reply by chemzqm** [reply]
You need find out the the folder of coc.nvim and use a basic vimrc looks like:
```vim
set nocompatible
set termguicolors
set runtimepath^=/path/to/coc.nvim
```
If you can't reproduce the issue, add other options and vim plugins
